ansible_rulebook/cli.py

```diff
@@ -279,6 +296,14 @@ def main(args: List[str] = None) -> int:
     if args.execution_strategy:
         settings.default_execution_strategy = args.execution_strategy
 
+    settings.websocket_address = args.websocket_address
+    settings.websocket_ssl_verify = args.websocket_ssl_verify
+    settings.websocket_token_address = args.websocket_token_address
```

Comment:
@bzwei Should we make this the URL I know the the previous one incorrectly uses address when its a url like ```ws://host.containers.internal:8001/api/eda/ws/ansible-rulebook
```